schunk_gripper_driver/schunk_gripper_driver/driver.py

```diff
@@ -683,6 +698,29 @@ def _move_to_absolute_position_cb(
         request: MoveToAbsolutePosition.Request,
         response: MoveToAbsolutePosition.Response,
         gripper: Gripper,
+    ):
+        self.get_logger().debug("---> Move to absolute position")
+        position = int(request.position * 1e6)
+        velocity = int(request.velocity * 1e6)
+        if self.needs_synchronize(gripper):
+            response.success = gripper["driver"].move_to_absolute_position(
+                position=position,
+                velocity=velocity,
+                use_gpe=False,
+                scheduler=self.scheduler,
+            )
+        else:
+            response.success = gripper["driver"].move_to_absolute_position(
+                position=position, velocity=velocity, use_gpe=False
+            )
+        response.message = gripper["driver"].get_status_diagnostics()
+        return response
+
+    def _move_to_absolute_position_with_gpe_cb(
```

Comment:
I would really love to have a single callback for the different variants. Else this becomes quite verbose. Any chance this could work with passing dictionaries (`*variable_args`) or a clever use of `partial`?